packages/gasket-redux/docs/redux-with-app-router.md

```diff
@@ -0,0 +1,142 @@
+# Redux with NextJS App Router
+
+## Overview
+
+Our current implementation of Redux in Gasket apps relies heavily on `getInitialProps` and the `_app.js` file which is not available in the App Router project structure. We use `next-redux-wrapper` in our current `@gasket/redux` and `@gasket/plugin-redux` packages. 
+It is now recommended to use the `@reduxjs/toolkit` in place of the `redux` package. In particular, the `createStore` method is deprecated, which we currently use in the `@gasket/redux` package. 
+
+In the new version of Gasket, we have decided to have the user setup Redux, instead of having a 
+package available.
+
+## Setup Guide
+
+Follow the steps below to add Redux to an existing Gasket application that is using the NextJS App Router architecture. 
+
+This guide assumes that a Gasket application has already been created using the `@gasket/preset-nextjs` preset. For details on how to do so please see the [quick start guide].  
+
+```bash
+create-gasket-app --presets @gasket/preset-nextjs
+```
+
+When prompted, make sure to select yes to using the App Router architecture.
+
+![App Router Prompt](./static/app-router-prompt.png)
```

Comment:
You're missing the screenshot image with your PR. Although, I'm not entirely sure it's necessary in this doc as your previous sentence is straight-forward enough.